src/content/tools/_index.md

```diff
@@ -21,11 +21,15 @@ The following types of tools are developed by NIST OSCAL team:
 
 In order to support data interchange and semantic interoperability in this heterogeneous environment, the OSCAL project publishes some basic software components along with the OSCAL specifications, to help serve as reference points or "scaffolding" for developers and for partners in data exchange.
 
-See the OSCAL  [XML](https://github.com/usnistgov/OSCAL/tree/main/xml) and [JSON/YAML](https://github.com/usnistgov/OSCAL/tree/main/json) resource pages in the project repository for more details.
+See the [OSCAL build resource page](https://github.com/usnistgov/OSCAL/blob/main/build/README.md) in the project repository for more details.
```

Comment:
Non-blocking: as discussed during standup, I am more than happy to accept this change and approve the PR as we will plan a "site release" of content from `develop` to `main` accordingly once we are ready for a core OSCAL release.